### PR TITLE
feat(procedures): gdpr implementation of document upload

### PR DESCRIPTION
**Pull request by oalkabouss**
ref: MANAGER-15339

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-15317`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | yes/no
| Tickets          | MANAGER-15339
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/procedures/src/pages/rgdp/rgdpForm/FileField/FileField.component.tsx
Comment (JacquesLarique) [inline packages/manager/apps/procedures/src/pages/rgdp/rgdpForm/FileField/FileField.component.tsx:1]: Wasn't it possible to reuse the `FormDocumentFieldList` component ?

File: packages/manager/apps/procedures/src/components/FileInput/FileInputList.tsx [hunk 1]
```diff
@@ -26,9 +27,11 @@ export const FileInputList: FunctionComponent<Props> = ({ className }) => {
...
     <div className={className}>
       {multiple && (
-        <label className="text-sky-900 text-xs font-semibold">
-          {t('account-disable-2fa-file-input-attachments')}
-        </label>
+        <OsdsText>
+          <span className="text-sky-900 text-xs font-semibold">
+            {t('account-disable-2fa-file-input-attachments')}
+          </span>
+        </OsdsText>
```
Comment (JacquesLarique) [inline packages/manager/apps/procedures/src/components/FileInput/FileInputList.tsx:34]: Why do you need the span inside the ODS text ?
Also, is there any impact on 2FA pages display ?

Comment (oalkabouss) [reply packages/manager/apps/procedures/src/components/FileInput/FileInputList.tsx:34]: i took the opportunity to add this text in the ods components, it has no impact on the display, as I compared the two

Comment (JacquesLarique) [reply packages/manager/apps/procedures/src/components/FileInput/FileInputList.tsx:34]: But why do we need a span inside the ODS text ? wouldn't the text component be enough ?

Comment (oalkabouss) [reply packages/manager/apps/procedures/src/components/FileInput/FileInputList.tsx]: because i wanted it to look as close as possible to the previous one

**Review by JacquesLarique — APPROVED** [review]

**Review by MaximeBajeux — APPROVED** [review]

**Pull request merged**